Packages/com.unity.inputsystem/InputSystem/Events/IMECompositionEvent.cs

```diff
@@ -89,50 +123,53 @@ public void Dispose()
             object IEnumerator.Current => Current;
         }
 
-        public int Count => size;
+        int m_Length;
+        /// <summary>
+        /// The number of characters in the current IME Composition
+        /// </summary>
+        /// <value>The Length of the Composition</value>
+        public int length { get { return m_Length; }}
 
+        IntPtr m_CharBuffer;
+
+
+        /// <summary>
+        /// An Indexer into an individual character in the IME Composition. Will throw an out of range exception if the index is greater than the length of the composition.
```

Comment:
Use `<exception`> tags to document exceptions.

Reply:
Done